RecoLocalCalo/Configuration/python/RecoLocalCalo_cff.py

```diff
@@ -31,3 +31,38 @@
 #
 from RecoLocalCalo.Configuration.hcalLocalRecoNZS_cff import *
 calolocalrecoNZS = cms.Sequence(ecalLocalRecoSequence+hcalLocalRecoSequence+hcalLocalRecoSequenceNZS) 
+
+def _modifyRecoLocalCaloConfigurationReconstructorsForPhase2CommonHBHE( obj ):
+    obj.digiLabel = cms.InputTag('simHcalDigis','HBHEUpgradeDigiCollection')
+
+def _modifyRecoLocalCaloConfigurationReconstructorsForPhase2CommonHO( obj ):
+    obj.digiLabel = cms.InputTag('simHcalDigis')
+
+def _modifyRecoLocalCaloConfigurationReconstructorsForPhase2CommonHF( obj ):
+    obj.digiLabel = cms.InputTag('simHcalDigis','HFUpgradeDigiCollection')
+
+def _modifyRecoLocalCaloConfigurationReconstructorsForPhase2CommonZDC( obj ):
+    obj.digiLabel = cms.InputTag('simHcalUnsuppressedDigis')
+
+def _modifyRecoLocalCaloConfigurationProcessForPhase2Common( theProcess ):
+    theProcess.load("RecoLocalCalo.HcalRecProducers.HBHEUpgradeReconstructor_cfi")
+    theProcess.load("RecoLocalCalo.HcalRecProducers.HFUpgradeReconstructor_cfi")
```

Comment:
My recommendation from Core is to avoid using `makeProcessModifier` unless there is no other way to achieve what is needed. I'm personally fine if that change is done now or happens in a later pull request, just as long as it does eventually happen. I don't wish to have these recommendations cause a needed change to be delayed.